lib/commands/create.js

```diff
@@ -131,13 +131,19 @@ module.exports = function (program) {
           ? '@apostrophecms/user:add'
           : 'apostrophe-users:add';
       let createUserCommand = `node app.js ${userTask} admin admin`;
-      
+
       // Prepend MongoDB URI if provided
-      if (options.uri) {
+      if (options.mongodbUri) {
         // Ensure the URI is properly quoted to handle special characters
-        createUserCommand = `APOS_MONGODB_URI="${options.uri}" ` + createUserCommand;
+        createUserCommand = `APOS_MONGODB_URI="${options.mongodbUri}" ` + createUserCommand;
```

Comment:
Use the `shell-quote` module to do this a little more safely now that we're talking about URIs from Atlas whose contents are beyond our control.

Reply:
Done.